includes/definitions/weos.yaml

```diff
@@ -0,0 +1,24 @@
+os: weos
+text: 'Westermo WeOS'
+type: network
+icon: westermo
+group: westermo
+ifname: true
+mib_dir: westermo
+over:
+    - { graph: device_bits, text: 'Device Traffic' }
+    - { graph: device_mempool, text: 'Memory Usage' }
+discovery:
+    - sysObjectID:
+        - .1.3.6.1.4.1.16177
+good_if:
+    - 1/1
+    - 1/2
+discovery_modules:
```

Comment:
Do you explicitely need to disabled these ? Do they fail/crash the device ? if  this is only because they are not supported, they will anyway not do much so you don't need to disable them

Reply:
Do you mean the discovery and poller modules? No, it's not necessary to explicitely disable them. I disabled them because the device doesn't support these protocols, but it won't cause any issues if they're enabled either. Should I enable them just to keep it simple?

Regarding the good_if, I added these because port 1/1 and 1/2 are the SHDSL ports, which by default are ignored by LibreNMS. I tried with **good_ifType: shdsl** at first, but it doesn't seem to work. Is it only possible to specify the **ifType** together with **bad_if** and not **good_if**?

Reply:
I have now removed the discovery_modules and poller_modules configuration.